app/views/shared/_navbar.html.erb

```diff
@@ -26,13 +26,12 @@
               <%= link_to "Home", "/", class: "nav-link" %>
             </li>
             <li class="nav-item">
-              <%= link_to "Messages", "#", class: "nav-link" %>
+              <%= link_to "My Bookings", "/reservations", class: "nav-link" %>
```

Comment:
/reservations should be reservations_path. In this case, it won't be in a string.